web/Team/progress/progress1.html

```diff
@@ -45,29 +45,95 @@
 		<!-- Instantiate a page banner image. Page banner images should be 1100x350px and should be located within the local
 			img folder located at this directory level. -->
 		<div id="banner">
-			<img src="Files/img/BannerImgExample.jpg"></img>
+			<img src="Files/img/BannerImgExample.jpg">
 		</div>
 	
 		<!-- Instantiate "tools" needed for a page. Tools are premade functional blocks that can be used to build a page,
 			and include things such as a file lister (for listing out homework assignments or tutorials)
 		-->
 		<div id="content">
             <h2>Progress Report for -insert student name here-</h2>
-            
-            <h4>Weeks 1-2:</h4>
+
+            <h4>Week 3:</h4>
             <b>Date:</b> -Enter today's date here (the date you posted)-<br>
             <b>Total hours:</b> -Enter the number of hours you worked this week here-<br>
             <b>Description of project design efforts:</b><br>
-            -Discuss your project design efforts here. Remember to follow the guidelines presented in the Progress Report Policy that is posted on Brightspace.</br>
+            -Discuss your project design efforts here. Remember to follow the guidelines presented in the Progress Report Policy that is posted on Brightspace.  This is a template entry; simply copy/paste this entry when doing progress reports each week.
 
+            <br>
             <br>
 
-            <h4>Week 3:</h4>
-            <b>Date:</b> -Enter today's date here (the date you posted)-<br>
-            <b>Total hours:</b> -Enter the number of hours you worked this week here-<br>
+            <h4>Weeks 1 & 2:</h4>
+            <b>Date:</b>Sep 1, 2023<br>
+            <b>Total hours:</b>16<br>
             <b>Description of project design efforts:</b><br>
-            -Discuss your project design efforts here. Remember to follow the guidelines presented in the Progress Report Policy that is posted on Brightspace.  This is a template entry; simply copy/paste this entry when doing progress reports each week.</br>
-                      
+            I have spent most of the past two weeks working on the design and
+            technical specifications of the kart. Most of this process was on

```

Comment:
Might want to revise `kart` to `table` or just `project` 😅.

Reply:
Omg, my brain is not working lmao, too many things going on XD